packages/@uppy/core/src/index.js

```diff
@@ -157,10 +166,8 @@ class Uppy {
       this.updateAll(nextState)
     })
 
-    // for debugging and testing
-    // this.updateNum = 0
+    // Exposing uppy object on window for debugging and testing
     if (this.opts.debug && typeof window !== 'undefined') {
-      window['uppyLog'] = ''
       window[this.opts.id] = this
```

Comment:
will be good that we'll be forced to get rid of this when we remove `debug: true` :eyes:

Reply:
Because exposing uppy should be done manually? probably not ideal to attach to window, but it helped me debug countless times.